src/deck-options.md

```diff
@@ -4,260 +4,221 @@
+before changing them, as mistakes can reduce Anki's effectiveness.
 
-Deck options are accessed by:
+If you're on the desktop app, do any of the following to open deck options:
 
-- Clicking the gear icon on the `Decks` screen.
-- Selecting a deck on the `Decks` screen, and then clicking `Options`
+- Click the gear icon on the Decks screen.
+- Select a deck on the Decks screen, and then click **Options**
   at the bottom of the screen.
-- Clicking on `More` > `Options` while in review mode.
-- Pressing `o` while in review mode.
+- Click "More" > "Options" while in review mode.
+- Press <kbd>o</kbd> while in review mode.
 
-This page describes the options shown in Anki 2.1.45+, when you have the v2 or
-v3 scheduler enabled. On older versions, some options will not be available, or
-will appear in a different section. Please keep in mind that the v1 scheduler
-is no longer supported in Anki 2.1.50+. If you have not yet updated to v2 or v3,
-you will be prompted to update when you attempt to review cards in 2.1.50+.
-
-For more info on deck options, please check:
+Here are some community posts about deck options:
 
 - [Deck Options Explained](https://forums.ankiweb.net/t/deck-options-explained/213)
 - [Deck Options in a Mental Map](https://forums.ankiweb.net/t/deck-options-in-a-mental-map/15757)
 
+Note: both of them were written for the earlier versions of Anki.
+
 ## Presets
 
 Anki allows you to share options between different decks, to make
-it easy to update options in many decks at once. To do this, options are
-grouped into _presets_. By default, all newly created decks use
-the same preset.
+it easier to update options in many decks at once. To do this, options are
+grouped into _presets_. If you change an option in a preset, the change is applied to all decks that use the same preset. All newly created decks use the "Default" preset.
 
-If you’d like to alter the settings on one deck but not other decks, click the
-arrow icon in the top right of the Deck Options window. The options are:
+To change the options in one deck but not other decks, click the
+arrow icon in the top right of the Deck Options window. The following are shown:
 
-- **Save**: Saves all modifications you've made since opening the deck options screen.
+- **Save**: Saves all modifications you've made after opening the deck options screen.
 - **Add**: Add a new preset, with the default options.
-- **Clone**: Clone your current present, which is useful if you
-  just want to modify certain options, keeping the rest as they are.
-- **Rename** Changes the name of the current preset.
-- **Delete** Deletes the current preset. This will require that the next sync is
+- **Clone**: Clone your current preset, which is useful if you
+   want to modify some of the  options but keep the rest as they are.
+- **Rename**: Changes the name of the current preset.
+- **Delete**: Deletes the current preset. This will make your next sync
```

Comment:
> but "require" can be really confusing in some contexts. Better to not use it. For one example, see [mailchimp].
I don't see how this is one of those situations where the meaning could go in either direction. We need a stronger word that "make" and I don't see a reason to change it from the original "require."

> That's pretty intuitive when it happens. Most people read docs for specific problems they're trying to solve. This wont be something most people will seek to learn in the manual.
I strongly disagree. First, I'm not sure how intuitive it is, since I'm sitting here right now unable to intuit it! 🙃 Second, "what happens when I delete this?" is definitely a question the docs should be able to answer.

Reply:
> We need a stronger word that "make" and I don't see a reason to change it from the original "require."

"X requires Y" can mean "If X happens, Y will happen" or it can mean "For X to happen, Y will need to happen". Sure, it's easy for you as a L1 speaker and experienced Anki user. I can make this "If you use this option, your next sync will be a one-way sync".

> Second, "what happens when I delete this?" is definitely a question the docs should be able to answer.

My point was we should be more task oriented and shouldn't fill up the manual with reference/concept that most users do not look for. Some of us may enjoy reading manuals, but most people aren't looking for this.

Reply:
> "X requires Y" can mean ...

Yes, it can, but as I said, that isn't an issue here. This is "X will require Y to be ..." -- the requirement flows both directions, so it's not ambiguous. 

> My point was we should be more task oriented

I still don't understand how this *is not* task oriented. It's about a specific command and what it causes to happen.

Reply:
> Yes, it can, but as I said, that isn't an issue here. This is "X will require Y to be ..." -- the requirement flows both directions, so it's not ambiguous.

Non-native English speakers, at least where I live, think of what immediately follows *require* as a precondition for what immediately precedes *requires* (which is how the word is used most of the time). That grammatical structure "X requires Y to be" also doesn't make sense in a lot of languages.

> I still don't understand how this is not task oriented. It's about a specific command and what it causes to happen.

I might have misinterpreted you. But I still think it's unnecessary information. Let's not do it here.